### ISSUE TITLE
[Feature Request] Prompt Styles Import

### ISSUE THREAD

**Issue by Seputaes** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#234]
It would be great if support for [`prompt-styles`](https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/22bcc7be428c94e9408f589966c2040187245d81/modules/api/api.py#L188) could be added to the plugin's UI. I realize that effectively the same thing can be achieved with the Prompts Library.

I maintain a large number of styles in the automatic1111 webui which I frequently use. Being able to select/append them in a similar way would greatly simplify workflows rather than needing to maintain them in 2 separate places or need to copy/paste between them.

**Comment by Wite905** [comment]
yes, it would be great, personally I don't use it practically only because there is no this function

**Comment by hakaserver** [comment]
I also wish the inclusion of this feature. I frequently use the prompt-styles function, together with multiple wildcards. Being able to simply select one of the style entries I created directly in photoshop would be a great addition.
